suorituspalvelu-ui/src/pages/_app.tsx

```diff
@@ -21,15 +23,18 @@ const queryClient = new QueryClient({
 
 export default function App({ Component, pageProps }: AppProps) {
   const config = use(configPromise);
+  const tolgee = use(tolgeePromise);
 
   return (
-    <QueryClientProvider client={queryClient}>
-      <SessionExpiredProvider>
-        <Layout>
-          <Component {...pageProps} />
-          <Script src={config.routes.yleiset.raamitUrl} />
-        </Layout>
-      </SessionExpiredProvider>
-    </QueryClientProvider>
+    <TolgeeProvider tolgee={tolgee}>
+      <QueryClientProvider client={queryClient}>
+        <SessionExpiredProvider>
```

Comment:
Näiden providereiden järjestyksellä ei varmaan oo oikeasti väliä mutta itse näkisin et ne kuuluu just päinvastaiseen järjestykseen koska queryclient ja tolgee varmaan riippuu sessioista mutta ei toisin päin